SPECS/glib-networking/glib-networking.spec

```diff
@@ -72,29 +68,42 @@ ninja test
 %defattr(-,root,root)
 
 %changelog
+* Sun Dec 12 2021 Chris Co <chrco@microsoft.com> - 2.59.1-8
```

Comment:
Shouldn't the author be you instead? And also with libmodulemd.spec

Reply:
My build environment broke over the weekend so chris helped out to make these changes for both libmodulemd and glib networking

Reply:
Got it.